assets/js/components/notifications/ZeroDataStateNotifications.stories.js

```diff
@@ -0,0 +1,231 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { provideModules, provideSiteInfo } from '../../../../tests/js/utils';
+import WithRegistrySetup from '../../../../tests/js/WithRegistrySetup';
+import { CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import { MODULES_ANALYTICS } from '../../modules/analytics/datastore/constants';
+import { provideAnalyticsMockReport } from '../../modules/analytics/util/data-mock';
+import { MODULES_SEARCH_CONSOLE } from '../../modules/search-console/datastore/constants';
+import { provideSearchConsoleMockReport } from '../../modules/search-console/util/data-mock';
+import ZeroDataStateNotifications from './ZeroDataStateNotifications';
+
+const searchConsoleArgs = {
+	startDate: '2021-08-18',
+	endDate: '2021-10-12',
+	dimensions: 'date',
+};
+
+const analyticsArgs = [
+	{
+		startDate: '2021-09-15',
+		endDate: '2021-10-12',
+		compareStartDate: '2021-08-18',
+		compareEndDate: '2021-09-14',
+		metrics: [
+			{
+				expression: 'ga:goalCompletionsAll',
+				alias: 'Goal Completions',
+			},
+			'ga:bounceRate',
+		],
+	},
+	{
+		startDate: '2021-09-15',
+		endDate: '2021-10-12',
+		compareStartDate: '2021-08-18',
+		compareEndDate: '2021-09-14',
+		dimensions: 'ga:date',
+		metrics: [
+			{
+				expression: 'ga:goalCompletionsAll',
+				alias: 'Goal Completions',
+			},
+			'ga:bounceRate',
+		],
+	},
+	{
+		startDate: '2021-09-15',
+		endDate: '2021-10-12',
+		compareStartDate: '2021-08-18',
+		compareEndDate: '2021-09-14',
+		metrics: [
+			{
+				expression: 'ga:users',
+				alias: 'Total Users',
+			},
+		],
+		dimensions: [ 'ga:channelGrouping' ],
+		dimensionFilters: { 'ga:channelGrouping': 'Organic Search' },
+	},
+	{
+		startDate: '2021-09-15',
+		endDate: '2021-10-12',
+		compareStartDate: '2021-08-18',
+		compareEndDate: '2021-09-14',
+		metrics: [
+			{
+				expression: 'ga:users',
+				alias: 'Total Users',
+			},
+		],
+		dimensions: [ 'ga:date' ],
+		dimensionFilters: { 'ga:channelGrouping': 'Organic Search' },
+	},
+	{
+		dimensions: [ 'ga:date' ],
+		metrics: [
+			{
+				expression: 'ga:users',
+			},
+		],
+		startDate: '2021-09-15',
+		endDate: '2021-10-12',
+	},
+];
+
+const Template = ( { setupRegistry } ) => (
+	<WithRegistrySetup func={ setupRegistry }>
+		<ZeroDataStateNotifications />
+	</WithRegistrySetup>
+);
+
+export const NoNotificationsAvailable = Template.bind( {} );
+NoNotificationsAvailable.storyName = 'No Notifications Available';
+NoNotificationsAvailable.args = {
+	setupRegistry: ( registry ) => {
+		provideSearchConsoleMockReport( registry, searchConsoleArgs );
+
+		for ( const options of analyticsArgs ) {
+			provideAnalyticsMockReport( registry, options );
+		}
+	},
+};
+
+export const AnalyticsGatheringData = Template.bind( {} );
+AnalyticsGatheringData.storyName = 'Analytics Gathering Data ';
```

Comment:
@hussain-t Let's remove the extra whitespace at the end of the string.
```suggestion
AnalyticsGatheringData.storyName = 'Analytics Gathering Data';
```